app/src/androidTest/java/com/google/android/stardroid/test/control/AstronomerModelWithMagneticVariationTest.java

```diff
@@ -90,6 +90,13 @@ public void testAssertVectorEquals() {
     }
     assertTrue(failed);
   }
+  
+  /**
+   * Convert from degrees to radians.
+   */
+  private static float radians() {
```

Comment:
This function needs to stay as it was with the argument.  It's converting angles in degrees to angles in radians, so it doesn't make sense to hard-code the 10 degrees.